corehq/apps/app_manager/views/modules.py

```diff
@@ -797,29 +807,55 @@ def overwrite_module_case_list(request, domain, app_id, module_unique_id):
                     src_module.case_type)
             )
         else:
-            _update_module_case_list(request, detail_type, src_module, dest_module)
-            dest_modules_msg.append(dest_module.default_name())
-            app.save()
-    _msg = _(f'Case list configuration updated from {src_module.default_name()} menu to {", ".join(map(str, dest_modules_msg))} menu(s).')
+            try:
+                _update_module_case_list(detail_type, src_module, dest_module, attrs_dict)
+                updated_modules.append(dest_module.default_name())
+                app.save()
+            except Exception as e:
+                logger.exception(f'Error in updating module: {dest_module.default_name()}', e)
```

Comment:
Use `notify_exception` here, which will send the error to sentry. No one monitors the web server logs.